tests/constants/label.constants.ts

```diff
@@ -57,13 +57,14 @@ export const LIGHT_BG_DARK_TEXT = {
 export const RESPONSE_REJECTED_LABEL = new Label(RESPONSE, RESPONSE_REJECTED, COLOR_RESPONSE_REJECTED);
 export const STATUS_DONE_LABEL = new Label(STATUS, STATUS_DONE, COLOR_STATUS_DONE);
 export const TYPE_DOCUMENTATION_BUG_LABEL = new Label(TYPE, TYPE_DOCUMENTATION_BUG, COLOR_TYPE_DOCUMENTATION_BUG);
+export const TYPE_FUNCTIONALITY_BUG_LABEL = new Label(TYPE, TYPE_FUNCTIONALITY_BUG, COLOR_TYPE_FUNCTIONALITY_BUG);
 
 export const SEVERITY_HIGH_LABEL = new Label(SEVERITY, SEVERITY_HIGH, COLOR_SEVERITY_HIGH);
 export const SEVERITY_MEDIUM_LABEL = new Label(SEVERITY, SEVERITY_MEDIUM, COLOR_SEVERITY_MEDIUM);
 export const SEVERITY_LOW_LABEL = new Label(SEVERITY, SEVERITY_LOW, COLOR_SEVERITY_LOW);
 
 // Constant array of labels to simulate Github response
-export const LABEL_ARRAY = [
+export const LABEL_ARRAY_TEAM_OR_MODERATION_PHASE = [
```

Comment:
```suggestion
export const ALL_LABEL_ARRAY = [
```

Same here

Reply:
I am not sure we can name it `ALL_LABEL_ARRAY` since this array only contains some labels involved in the dev response and moderation phase. I think it would be just a bit misleading to name it as `ALL_LABEL_ARRAY`

Reply:
Maybe I will just revert it back to `LABEL_ARRAY`

Reply:
okay sounds good as well @ducbinh2611